test/addons/errno-exception/test.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const binding = require(`./build/${common.buildType}/binding`);
+const err = binding.errno();
+assert.strictEqual(err.syscall, 'syscall');
```

Comment:
Nit time (from [the guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure))...

```js
'use strict';

const common = require('../../common');

// Verify that the path argument to node::ErrnoException() can contain UTF-8
// characters.

const assert = require('assert');
const binding = require(`./build/${common.buildType}/binding`);
const err = binding.errno();

assert.strictEqual(err.syscall, 'syscall');
assert.strictEqual(err.errno, 10);
assert.strictEqual(err.path, 'päth');
```